include/nuttx/power/battery_ioctl.h

```diff
@@ -143,6 +143,7 @@ enum batio_operate_e
   BATIO_OPRTN_SYSON,
   BATIO_OPRTN_RESET,
   BATIO_OPRTN_WDOG,
+  BATIO_OPRTN_SHIPMODE,
```

Comment:
I was going to suggest a generic name like _STANDBY, but I agree _SHIPMODE is has a sense of purpose. In the automotive industry we use it too when shipping a car from a country to another or even inside big countries

Reply:
The standby is too general and may give the programmer the expression that the charger will switch between standby and normal frequently. But the ship mode is set on factory line, exit when the customer activates the device at first time, and then never come back to the ship mode.